charts/cass-operator-chart/templates/customresourcedefinition.yaml

```diff
@@ -6307,9 +6307,7 @@ spec:
               type: string
           type: object
       type: object
-      {{- if semverCompare ">= 1.15-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
I think we need to keep this if we want to support k8s <= 1.14 - which we might be OK moving away from

Reply:
@jimdickinson should I put the conditional back?

Reply:
for this PR, I think so... I thought we had some more hackery around getting `x-kubernetes-preserve-unknown-fields: true` into the yaml, but maybe that had already been cleaned up and turned into "apply the patch"